docs/framework/network-programming/enabling-and-disabling-ipv6.md

```diff
@@ -23,10 +23,10 @@ To use the IPv6 protocol, ensure that you are running a version of the operating
  The .NET Framework has a configuration switch set in a configuration file as follows  
   
 ```xml  
-<system.net>…  
-    <settings>…  
-        <ipv6 enabled="true"/>…  
-    </settings>…  
+<system.net>...  
```

Comment:
Nit: I would put each `...` on new line, unless it is in the form `<elem>...</elem>`.